runtime/polkadot/src/lib.rs

```diff
@@ -1085,11 +1064,32 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllModules,
-	BabeEpochConfigMigrations,
+	(BabeEpochConfigMigrations, KillOffchainPhragmenStorage),
 >;
 /// The payload being signed in transactions.
 pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
 
+parameter_types! {
+	pub const StakingPrefix: &'static str = "Staking";
```

Comment:
We can get this from palletInfo? Probably yes. But it is trivial as well.

Reply:
actually staking is still not using pallet info for pallet prefix. it is hardcoded as `"Staking"` in decl_storage call